src/core/trtserver.h

```diff
@@ -280,6 +287,23 @@ TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_ServerOptionsSetServerId(
 TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_ServerOptionsSetModelRepositoryPath(
     TRTSERVER_ServerOptions* options, const char* model_repository_path);
 
+// Set the model control mode in a server options. For each mode the models
+// will be managed as the following:
+//
+//   TRTSERVER_MODEL_CONTROL_NONE: the models in model repository will be loaded
+//   on startup. The model states will not be modified after startup.
```

Comment:
Change the last sentence to: "After startup any changes to the model repository will be ignored. Calling TRTSERVER_ServerPollModelRepository will result in an error."